### PR TITLE
HARP-8442 Enable mixed lod for all examples with globe support.

### PR DESCRIPTION
**Pull request by MPursche**
Signed-off-by: Marcel Pursche <marcel.pursche@here.com>

Thank you for contributing to harp.gl!

Before requesting a pull request, please remember to check the following documents:
* [contribution guidelines](https://github.com/heremaps/harp.gl/blob/master/CONTRIBUTING.md)
* [coding style](https://github.com/heremaps/harp.gl/blob/master/CODINGSTYLE.md)

If you are adding new functionality we would highly appreciate if you can describe what is the capability you are adding and even better if you can add some examples. Please also remember to add tests for it.

# CI Check

Our bots will check whether your PR can be directly integrated into the mainline. We have some internal integration tests running on the background, our bots will inform you of the next steps and someone from our team will take a look and help if needed!

And please do not forget to sign-off your commit! You can read more about DCO [here](https://julien.ponge.org/blog/developer-certificate-of-origin-versus-contributor-license-agreements/). But, in short, you just need to use `git commit -s` or append `--signoff` when you are committing to the repo.

Happy contributing!

### CODE REVIEW

File: @here/harp-examples/src/datasource_satellite-tile.ts [hunk 1]
```diff
@@ -33,7 +33,8 @@ export namespace SatelliteDataSourceExample {
 
         const map = new MapView({
             canvas,
-            theme: "resources/berlin_tilezen_base_globe.json"
+            theme: "resources/berlin_tilezen_base_globe.json",
+            enableMixedLod: true
```
Comment (ninok) [inline @here/harp-examples/src/datasource_satellite-tile.ts]: I wonder if we should enable mixed LOD by default if `projection===sphereProjection`

Comment (MPursche) [reply @here/harp-examples/src/datasource_satellite-tile.ts]: For this example it wouldn't work as it starts with `mercartorProjection` and can be switched to `sphereProjection`. The `enableMixedLod` would need to be changed every time `projection` is changed + some kind of override to allow to change it explicitly.

File: @here/harp-mapview/lib/MapView.ts [hunk 2]
```diff
@@ -1047,11 +1047,11 @@ export class MapView extends THREE.EventDispatcher {
...
     }
 
-    get enableMixedLod(): boolean {
+    get enableMixedLod(): boolean | undefined {
         return this.m_enableMixedLod;
     }
 
-    set enableMixedLod(enableMixedLod: boolean) {
+    set enableMixedLod(enableMixedLod: boolean | undefined) {
         this.m_enableMixedLod = enableMixedLod;
```
Comment (CraigBerlin) [inline @here/harp-mapview/lib/MapView.ts]: Skip update if the old value equals the new value?

File: @here/harp-mapview/lib/MapView.ts [hunk 3]
```diff
@@ -963,10 +964,9 @@ export class MapView extends THREE.EventDispatcher {
                 ? new PhasedTileGeometryManager(this)
                 : new SimpleTileGeometryManager(this);
 
-        this.m_enableMixedLod = getOptionValue(
-            options.enableMixedLod,
-            MapViewDefaults.enableMixedLod
```
Comment (CraigBerlin) [inline @here/harp-mapview/lib/MapView.ts]: Can we now remove this from MapViewDefaults?

**Pull request merged**